### DISCUSSION TITLE
Where do I add the vault and repo details?

### DISCUSSION THREAD

**Discussion by ronykris** [Vinzent03/obsidian-git#115] [Q&A]
Hi there,

Where do I mention the vault and repo details for backup? I couldn't find anything similar in the settings.

Any help is appreciated.

Thank you.

**Comment by Vinzent03** [comment]
Sorry, I don't really understand your question. Do you mean where you set for example the github url?

**Reply by ronykris** [reply]
So, I got the plugin installed and want to backup one of my vaults to a specific repo. Where do I specify the vault and the repo?

**Reply by ronykris** [reply]
Can you share a screenshot if possible?

**Reply by Vinzent03** [reply]
So I guess you haven't set up a local repo yet. Please run the command "Initialize new repo". Follow this [guide](https://docs.github.com/en/repositories/creating-and-managing-repositories/creating-a-new-repository) to create a new GitHub repo. After that, run the command "Create Backup". Make sure your credentials are set up, but that depends most of the time on user and OS.

**Reply by ronykris** [reply]
Thank you. Looks like I cannot use an existing repo. Is that correct?

Also, where do I execute the create backup command? On the os terminal?

**Reply by Vinzent03** [reply]
Well, you could have explained that. I cannot help you correctly if you don't explain your complete situation. It's of course possible to use an existing repo. But therefore you have to run `git clone <your-repo-url>` in the terminal. https://docs.github.com/en/get-started/getting-started-with-git/about-remote-repositories This will download the repo.  Then you have to choose that folder as vault in Obsidian.
The command would be in Obsidian.
